Improving aws costs docs (#229)
FairwindsOps/insights-docs: docs/technical-details/reports/aws-costs.md

it can know precisely what you're spending on nodes and use that
information to infer accurate workload costs.

> AWS Costs Report is built on [AWS costs and Usage Report](https://docs.aws.amazon.com/cur/latest/userguide/what-is-cur.html).

Reviewer: ```suggestion
The AWS Costs Report is built on [AWS costs and Usage Report](https://docs.aws.amazon.com/cur/latest/userguide/what-is-cur.html).
```

> First step is to create the Athena infrastructure using Terraform, CloudFormation, etc. The CUR report is created by AWS and stored in AWS S3.

Reviewer: ```suggestion
The first step is to create the Athena infrastructure using Terraform, CloudFormation, etc. The CUR report is created by AWS and stored in AWS S3.
```

> Basically on the Athena process AWS collects CUR data from S3 and make it available as a SQL table that can be queried.

Reviewer: ```suggestion
The Athena process AWS collects CUR data from S3, and makes it available as a SQL table that can be queried.
```

> "Athena is out-of-the-box integrated with AWS Glue Data Catalog".
> If you go to AWS Glue you can see there the infrastructure previously created to connect S3 CUR data into Athena.

Reviewer: ```suggestion
If you go to AWS Glue you can see there the infrastructure previously created to connect S3 CUR data into Athena.
```

Not sure where that quote came from...lmk if we should leave it in, or if we should link to the AWS docs

  workgroup: cur_athena_workgroup

  # tagkey is the key used to tag your nodes based on which cluster they belong to
  tagkey: kubernetes_cluster
  # tagvalue is the value used for this particular cluster
  tagvalue: staging
```

* **database**: the database created on AWS Glue Data
* **table**: aws cur report name
* **tagkey**: tag key is the tag used on EC2 to indicate that it's a cluster node. Ex: KubernetesCluster (in case of Kops). The column name in Athena has a prefix resource_tags_user_. Also AWS applies pascal camel to split the tag name. In this example the column in Athena will be: resource_tags_user_kubernetes_cluster.
* **tagvalue**: the value associated to the tag for filtering. Ex: production, staging
* **catalog**: default AWS Glue Catalog is AwsDataCatalog

> workgroup: workgroup created on Athena to be used on querying

Reviewer: ```suggestion
* **workgroup**: workgroup created on Athena to be used on querying
```
